sbc_compassion/models/import_letter_line.py

```diff
@@ -27,6 +27,8 @@ class ImportLetterLine(models.Model):
                                      compute='_set_sponsorship_id')
     sponsorship_status = fields.Boolean(compute='_set_sponsorship_id',
                                         readonly=True)
+    person = fields.Boolean(compute='_set_sponsorship_id',
```

Comment:
It is for the status of the partner or the child (thus not a sponsorship), but I did not spend enough time to think about the name